microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -0,0 +1,145 @@
+{% extends "sitebase.jinja2" %}
+{% set page_title = "Sample Results" %}
+{% set show_breadcrumbs = True %}
+
+{% block head %}
+
+<style>
+    /*Removes the .active, .accordion:hover effect from minimal_interface.css*/
+    /* Could potentially use color: initial but has less support...? */
+    .tab-pane {
+        background-color: #FFFFFF;
+        color: #000000;
+    }
+</style>
+
+<script type="text/javascript" language="javascript" src="/static/js/ruleset.js"></script>
+<script>
+    populate_select = function(select_selector, value_to_text, active_value){
+        var select = $(select_selector)
+        select.empty()
+        for (var v in value_to_text)
+        {
+            var option = $('<option></option>')
+                .attr("value", v)
+                .text(value_to_text[v]);
+
+            if (v == active_value)
+                option.attr('selected','selected');
+
+            select.append(option)
+        }
+    }
+
+    updateCompare = function(compareData){
+        //TODO
+        $("#compare").text(compareData["result"])
+    }
+    updateDiversity = function(diversityData){
+        //TODO
+        $("#diversity").text(diversityData["result"])
+    }
+    updateSimilarity = function(similarityData){
+        //TODO
+        $("#similarity").text(similarityData["result"])
+    }
+    updateTaxonomy = function(taxonomyData){
+        //TODO
+        $("#taxonomy").text(taxonomyData["result"])
+    }
+
+    init = function(){
+        // TODO: Should data type options be determined by ajax call or
+        // passed into jinja2 template?
```

Comment:
Probably ajax as this is coming from the public api. If a sample was only subject to one data type, then the pulldown shouldn't be selected